og.module

```diff
@@ -201,6 +215,11 @@ function og_og_membership_type_access(OgMembershipTypeInterface $entity, $operat
 function og_entity_create_access(AccountInterface $account, array $context, $bundle) {
   $entity_type_id = $context['entity_type_id'];
 
+  if (!empty($context['skip_og_permission'])) {
```

Comment:
This workaround `skip_og_permission` indicates a flaw in the logic of this PR. We should not introduce a bypass in access checking. This will just lead to more complexity down the road and potential security vulnerabilities. There must be a loop introduced somewhere, where an access check is done inside of an access check. We should fix this in a different way.